src/plugins/vis_type_vega/public/vega_view/vega_base_view.js

```diff
@@ -93,7 +92,7 @@ export class VegaBaseView {
     this._initialized = true;
 
     try {
-      this._$parentEl.empty().addClass(`vgaVis`).css('flex-direction', this._parser.containerDir);
+      this._parentEl.empty().addClass(`vgaVis`).css('flex-direction', this._parser.containerDir);
```

Comment:
empty, addClass, css methods do not exist on pure DOM.

Reply:
Many thanks for the review. Would something like this work?

`      this._parentEl.classList.add('vgaVis'); `
 `     this._parentEl.style.flexDirection = this._parser.containerDir;`